### DISCUSSION TITLE
Traits

### DISCUSSION THREAD

**Discussion by alexandeScott** [Truinto/DarkCodex#99] [General]
I love this!!!! 
However I have an issue, General.CreateBackgrounds I cant find the two traits. I have a check mark next to backgrounds. I can see the feat additional traits. 

But I don't see the two new traits in either backgrounds or available under additional traits. 

What am I doing wrong? 
thanks

**Comment by Truinto** [comment]
Is in scholar group. You cannot pick groups you already have picked from. If you want to circumvent that, Toybox has options for it.

**Comment by alexandeScott** [comment]
THANKS 
:) 


-----Original Message-----
